rpc/internal/service.go

```diff
@@ -68,38 +68,68 @@ func (srv *perfService) AttachResultData(ctx context.Context, result *ResultData
 		record.Artifacts = append(record.Artifacts, *i.Export())
 	}
 
+	if result.Rollups != nil {
+		rollups, err := result.Rollups.Export()
+		if err != nil {
+			return nil, errors.Wrap(err, "problem getting rollups")
+		}
+		record.Rollups = &rollups
+	}
```

Comment:
is there are rollups in the results, shouldn't we do the Add thing?

Reply:
Created `addRollups` helper function